crates/solver/src/liquidity.rs

```diff
@@ -90,40 +90,50 @@ pub enum Exchange {
 }
 
 /// Used to differentiate between different types of orders that can be sent to solvers.
-/// User orders containing OrderUid are the orders from the orderbook.
+/// User orders (market + limit) containing OrderUid are the orders from the orderbook.
 #[derive(Debug, Clone)]
 #[cfg_attr(test, derive(Derivative))]
 #[cfg_attr(test, derivative(PartialEq))]
 pub enum LimitOrderUid {
```

Comment:
AFAIU this is used as an ID for `LimitOrder`s in the `solver`. With that in mind, I think that "ID" should appear somewhere in the name.

Extremely nit-picky, but I would actually call it `*Id` (since, AFAIK, it isn't required for uniquely identifying orders, but just some debugging metadata):
```rust
enum LimitOrderId {}
enum LiquidityOrderId {}
```

Reply:
renamed